examples/astro-keystone/astro.config.mjs

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'astro/config';
+import vercel from '@astrojs/vercel/serverless';
+
+// https://astro.build/config
+export default defineConfig({
+  output: 'server',
+  adapter: vercel(),
```

Comment:
Maybe we shouldn't have this `adapter` for the example?

Reply:
Agreed, or use the node adapter - was mainly seeing if it made a difference to the Prisma error I am getting. I think an adapter is only required to "deploy" - https://docs.astro.build/en/guides/server-side-rendering/#adding-an-adapter